packages/flutter_tools/templates/app/android.tmpl/app/src/main/res/values/styles.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <!-- Theme applied to the Android Window while the process is starting -->
-    <style name="LaunchTheme" parent="@android:style/Theme.Black.NoTitleBar">
+    <style name="LaunchTheme" parent="@android:style/Theme.Light.NoTitleBar">
```

Comment:
ummm this is reasonable. Thanks for catching this

Reply:
Thanks for review! Could you please check again?